summary_workflows/differential_usage/specification/example_files/aggregation_out.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Can you make this a list of 1 aggregation object? ( I mean just square brackets at beginning and end of file)